web/views/execution_history.erb

```diff
@@ -0,0 +1,17 @@
+<table class="table">
+  <thead>
+    <tr>
+      <th>event</th>
+      <th>time</th>
+      <th>world</th>
+    </tr>
+  </thead>
+
+<% @plan.execution_history.each do |event| %>
+  <tr>
+    <td><%= h(event.name) %></td>
+    <td><%= h(Time.at(event.time).utc) %></td>
+    <td><%= h(event.world_id) %></td>
+  </tr>
+<% end %>
+</table>
```

Comment:
No: we need to keep the number of unusual concepts on some reasonable level (not advocating erb, purely pragmatic reasons)